clients/src/main/java/org/apache/kafka/common/utils/AppInfoParser.java

```diff
@@ -57,7 +56,7 @@ public static void registerAppInfo(String prefix, String id) {
             AppInfo mBean = new AppInfo();
             ManagementFactory.getPlatformMBeanServer().registerMBean(mBean, name);
         } catch (JMException e) {
-            log.warn("Error registering AppInfo mbean", e);
+            log.warn("Error registering AppInfo mbean: {}", e.getMessage());
```

Comment:
That seems pretty arbitrary though since you just agreed that users should be worried either way. The problem may be that people are using warn when they should use info.